daemon/api.go

```diff
@@ -958,7 +959,34 @@ func changeInterfaces(c *Command, r *http.Request) Response {
 		if len(a.Plugs) == 0 {
 			return BadRequest("at least one plug is required")
 		}
-		err := c.d.interfaces.AddPlug(&a.Plugs[0])
+		// NOTE: This constructs a partial snap.yaml meta-data. Later on it
+		// should reference real meta-data so that it chimes in with real
+		// snaps. Alternatively, this action should be removed.
```

Comment:
@mvo5 yes, we could, but as @niemeyer said we'll probably drop those actions this week so I wouldn't bother. Those were experimental commands and it will cost us more and more to maintain the illusion that we can add a plug or slot to the system.